fe1-web/src/features/rollCall/screens/RollCallOpened.tsx

```diff
@@ -105,7 +105,7 @@ const RollCallOpened = () => {
 
   const onCloseRollCall = () => {
     const attendeesList = Array.from(attendees).map((key: string) => new PublicKey(key));
-    return requestCloseRollCall(rollCallID, attendeesList)
+    return requestCloseRollCall(rollCallIDAlias, attendeesList)
```

Comment:
Are you sure this works as intended ? I'm concerned because I'm pretty sure you need to close the roll call using the base `rollCallID` and not its alias.

Reply:
The message API actually expects us to use the idAlias because it uses it to create a new updateId, which depends on the last idAlias